--- a/scripts/002_SDL_image-1.2.10.sh
+++ b/scripts/002_SDL_image-1.2.10.sh
@@ -11,7 +11,13 @@
 ./configure --prefix="$PS3DEV/portlibs/ppu" --host=powerpc64-ps3-elf \
 	--disable-sdltest \
 	--with-sdl-exec-prefix="$PS3DEV/portlibs/ppu" \
+	--without-x \
+	--disable-imageio \
 	--disable-shared \
+	--disable-jpg-shared \
+	--disable-png-shared \
+	--disable-tif-shared \
+	--enable-static \
 	LIBPNG_CFLAGS="`$PS3DEV/portlibs/ppu/bin/libpng-config --cflags`" \
 	LIBPNG_LIBS="`$PS3DEV/portlibs/ppu/bin/libpng-config --libs`" \
     	|| { exit 1; }
